backend/init_data/skills/pptx/pptx_tool.py

```
@@ -0,0 +1,347 @@
+    async def _emit_progress(self, status: str, message: str) -> None:
+        """Emit progress via WebSocket."""
+        if self.ws_emitter:
+            try:
+                await self.ws_emitter.emit_tool_call(
+                    task_id=self.task_id,
+                    tool_name=self.name,
+                    tool_output={"status": status, "message": message},
+                    status=status,
+                )
+            except Exception:
+                pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)